ambari-server/src/main/java/org/apache/ambari/server/state/SingleSignOnInfo.java

```diff
@@ -111,7 +119,7 @@ public void setSupported(Boolean supported) {
   /**
    * Gets the configuration specification that can be used to determine if SSO has been enabled or not.
    *
-   * @return a configuration specification (config-type/property_name)
+   * @return a configuration specification (JSON predicate)
    */
   public String getEnabledConfiguration() {
```

Comment:
A separate property can return the Predicate, rather than _compiling_ it over and over again.